client/components/SampleCensus.js

```diff
@@ -34,35 +77,35 @@ const QuestionAnswerBox = ({title, text}) => (
         padding: '5px',
```

Comment:
why is this style here and not in a scss file?

Reply:
Should be fixed in #88